### PR TITLE
k6 v0.45.0 updates

### PR DESCRIPTION
**Pull request by mstoykov**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **types/k6/execution.d.ts#L99** - The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d55e13d214cd08482ed636781d53c67e7425870a
</p>

File: types/k6/options.d.ts [hunk 1]
```diff
@@ -439,3 +442,8 @@ export type Scenario =
     | ConstantArrivalRateScenario
     | RampingArrivalRateScenario
     | ExternallyControlledScenario;
+
+export interface ScenarioOptions {
+    /** Browser specific options */
+    browser?: any;
```
Comment (imiric) [inline types/k6/options.d.ts:448]: Will the browser team add definitions here? Otherwise it's not very useful.

File: types/k6/test/options.ts [hunk 2]
```diff
@@ -101,3 +101,28 @@ const tlsOptions4: Options = {
...
     ],
 };
+
+const browserScenarios: Scenario[] = [
+    {
+        executor: 'shared-iterations',
+        iterations: 100,
+        options: {
+            browser: {
+                type: 'chromum',
```
Comment (imiric) [inline types/k6/test/options.ts:111]:
```suggestion
                type: 'chromium',
```

**Review by imiric — APPROVED** [review]
I don't use TS with k6, and am not very familiar with the definition syntax, but this LGTM.

Someone more familiar should give the final approval, though.

File: types/k6/options.d.ts (hunk 1 above)
Comment (mstoykov) [reply types/k6/options.d.ts:448]: At this point this is the definition that I am aware of. 

Also AFAIK the whole k6/experimental/browser needs rework after the currently ongoing rewrite.

**Review by codebien — APPROVED** [review]
I'm also not very familiar with TS but the current docs sounds reasonable to me. LGTM.

**Comment by mstoykov** [comment]
Ready to merge

**Pull request merged**